scripts/requirements.txt

```diff
@@ -32,3 +32,4 @@ toml==0.10.2
 urllib3==1.26.5
 websocket-client==1.2.1
 analytics-python==1.4.0
+pynacl==1.5.0
```

Comment:
We only need this if you plan to run the setup python steps before running the script, eg: 


          python3 -m venv ve1
          cd scripts && ../ve1/bin/pip3 install -r requirements.txt && cd ..
          cd scripts && ../ve1/bin/python3 setup.py install && cd ..

Reply:
Yes correct. Since we are not running the script from workflow, it is not necessary. Reverting this change.